quarkus/pom.xml

```diff
@@ -8,15 +8,14 @@
   <properties>
     <surefire-plugin.version>2.22.2</surefire-plugin.version>
     <maven.compiler.target>11</maven.compiler.target>
-    <quarkus.platform.version>2.6.3.Final</quarkus.platform.version>
     <maven.compiler.source>11</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.compiler.parameters>true</maven.compiler.parameters>
-    <quarkus-plugin.version>2.7.0.Final</quarkus-plugin.version>
     <compiler-plugin.version>3.9.0</compiler-plugin.version>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
```

Comment:
Once 2.7.0.Final is officially out next week, I recommend you to change that to `io.quarkus.platform`.

Reply:
it came from dependabot https://github.com/ozkanpakdil/test-microservice-frameworks/pull/350 , why it is bringing 2.7.0 without official release, strange.

Reply:
It's because you are targeting the core BOM. For `*.0.Final` releases, we release the core artifacts 1 week before the official release to get the Platform members the time to update. Then we release the Platform and we announce the release.

So `io.quarkus:quarkus-bom:2.7.0.Final` is already released but `io.quarkus.platform:quarkus-bom` will be released next Wednesday.
We now recommend to use the Platform one (and the Platform Maven plugin). That's what you get now if you create new projects on code.quarkus.io.